prow/config/config.go

```diff
@@ -381,11 +381,29 @@ type Plank struct {
 	// DefaultDecorationConfig are defaults for shared fields for ProwJobs
 	// that request to have their PodSpecs decorated
 	DefaultDecorationConfig *prowapi.DecorationConfig `json:"default_decoration_config,omitempty"`
```

Comment:
Could we add to the GoDoc to say it's deprecated and to use `'*'` instead

Reply:
addressed